pkg/resources/data.go

```diff
@@ -534,7 +534,7 @@ func (d *TemplateData) NodeLocalDNSCacheEnabled() bool {
 func (d *TemplateData) KubermaticAPIImage() string {
 	apiImageSplit := strings.Split(d.kubermaticImage, "/")
 	var registry, imageWithoutRegistry string
-	if len(apiImageSplit) != 3 {
+	if len(apiImageSplit) < 3 {
 		registry = RegistryDocker
```

Comment:
I'm not sure should we actually default the registry ourselves. Usually, that's done by the container runtime, which may have a default registry that's not `docker.io`.